ipinfo/cmd_tool.go

```diff
@@ -26,7 +28,8 @@ func printHelpTool() {
 
 Commands:
   aggregate    aggregate IPs, IP ranges, and CIDRs.
-
+  lower        get start IP of CIDR
+  upper        get end IP of CIDR
 Options:
```

Comment:
Need a newline above this for better separation.

Reply:
@Kashaf-Sajid-Yaseen this was not resolved, rather it was resolved incorrectly. I meant above `Options`